frontend/lib/norent/letter-builder/confirmation.tsx

```diff
@@ -1,26 +1,98 @@
 import React, { useContext } from "react";
 import Page from "../../ui/page";
 import { AppContext } from "../../app-context";
+import { CenteredPrimaryButtonLink } from "../../ui/buttons";
+import { OutboundLink } from "../../analytics/google-analytics";
+
+const checkCircleSvg = require("../../svg/check-circle-solid.svg") as JSX.Element;
 
 export const NorentConfirmation: React.FC<{}> = () => {
   const { session } = useContext(AppContext);
   const letter = session.norentLatestLetter;
+  const user = session.norentScaffolding;
 
   return (
-    <Page
-      title="Your letter has been sent!"
-      withHeading="big"
-      className="content"
-    >
-      <p>Noice.</p>
-      {letter?.trackingNumber ? (
+    <Page title="Your letter has been sent!" className="content">
+      <div className="media">
```

Comment:
Sadly, the `.jf-heading-with-icon` class was not quite what Tahnee wanted on the confirmation page here (see designs for mobile), but luckily it was super easy to implement this configuration instead with the Bulma `.media` class.

Reply:
AW

can u file an issue to maybe make it more standardized across our sites, once we finish this sprint?  it feels so similar that it seems like it oughtta be the same doggone react component...